### ISSUE TITLE
[Bug] `update_snapshots` doesn't save at TF state with `mongodbatlas_cloud_backup_schedule` resource

### ISSUE THREAD

**Issue by ivan-sukhomlyn** [mongodb/terraform-provider-mongodbatlas#904]
After the initial creation of the MongoDB Atlas Backup policy for the On-Demand cluster, I experienced the situation of TF resource change triggering with `terraform plan` without modifications of the `mongodbatlas_cloud_backup_schedule` Terraform resources.

The Terraform provider always tries to update the resources with modification from default `false` to `true` described in the config.

### Terraform CLI and Terraform MongoDB Atlas Provider Version

```hcl
>> terraform version
Terraform v1.2.9
on darwin_arm64
+ provider registry.terraform.io/mongodb/mongodbatlas v1.4.6
```

### Terraform Configuration File 

```hcl
### MongoDB Atlas Cloud Backups
resource "mongodbatlas_cloud_backup_schedule" "product" {
  count = var.mongo_atlas_cloud_backup_enabled ? 1 : 0

  project_id   = data.mongodbatlas_project.product.id
  cluster_name = mongodbatlas_advanced_cluster.product.name

# ...

  update_snapshots    = true
  auto_export_enabled = false
}
```

### Steps to Reproduce
Without any changes to the config after initial apply

1. `terraform init`
2. `terraform apply` or `terraform plan`
3. plan result:

```hcl
Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # mongodbatlas_cloud_backup_schedule.product[0] will be updated in-place
  ~ resource "mongodbatlas_cloud_backup_schedule" "product" {
        id                                       = "Y2x1c3Rlcl9uYW1l:cHJvZHVjdC1hcGktcHJvZA==-cHJvamVjdF9pZA==:NjM1OGY4OTRmNDdmYzAxOGFiMmFkN2Jk"
      ~ update_snapshots                         = false -> true
        # (10 unchanged attributes hidden)

        # (2 unchanged blocks hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.
```

### Expected Behavior
`update_snapshots` applies only with modification of the `mongodbatlas_cloud_backup_schedule` resource and doesn't trigger **update in-place** action.

### Actual Behavior
`update_snapshots` attribute triggers TF state updates without chvanages to the `mongodbatlas_cloud_backup_schedule`.

**Comment by martinstibbe** [comment]
@ivan-sukhomlyn able to reproduce same issue added to [INTMDB-472](https://jira.mongodb.org/browse/INTMDB-472) to resolve issue

**Comment by Zuhairahmed** [comment]
@ivan-sukhomlyn this issue has been picked up in current sprint cycle to investigate further and should have an update for you next week.

**Comment by ivan-sukhomlyn** [comment]
The issue has been fixed with the [v1.8.0](https://github.com/mongodb/terraform-provider-mongodbatlas/releases/tag/v1.8.0) version.

Thank you, @Zuhairahmed @martinstibbe!

**Comment by Zuhairahmed** [comment]
glad to hear it, thanks @ivan-sukhomlyn!
